src/HelpersServiceProvider.php

```diff
@@ -19,10 +20,14 @@ public function boot()
         $router->prependMiddlewareToGroup('api', SecurityMiddleware::class);
 
         Validator::extend('unique_except_of_authorized_user', function ($attribute, $value) {
+            if (gettype($value) !== 'string') {
+                throw new UnprocessableEntityHttpException("The {$attribute} must be a string.");
+            }
+
             $userId = Auth::id();
             $result = DB::select("select count(*) as entities_count from users where id <> {$userId} and {$attribute} = '{$value}';");
```

Comment:
let's get $keyField from the second attribute with the default `id` value

also please use bindings to prevent the sql injections

```suggestion
            $result = DB::select("select count(*) as entities_count from users where {$keyField} <> ? and {$attribute} = ?;", [
                $userId,
                $value
            ]);
```